### PR TITLE
Create hf03614-burlington.md

### PR DESCRIPTION
**Pull request by hflora**
Added places to eat and things to do.

### CODE REVIEW

File: places/hf03614-burlington.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### Places to Eat
+- The works
+- Shoeless joes
+- Bhan thai
+- Emmas back porch
+
```
Comment (noelvaneynde-exclaim) [inline places/hf03614-burlington.md:6]: Sounds like the food at Emma's is really good.

File: places/hf03614-burlington.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+### Places to Eat
+- The works
+- Shoeless joes
+- Bhan thai
```
Comment (noelvaneynde-exclaim) [inline places/hf03614-burlington.md:4]: I love curry.

Comment (ravno) [inline places/hf03614-burlington.md:4]: Oh, I like Thai food. @hflora Do they also cook spice ?

File: places/hf03614-burlington.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+### Places to Eat
+- The works
+- Shoeless joes
```
Comment (TedSpinks) [inline places/hf03614-burlington.md:3]: funny name!

**Comment by hflora** [comment]
@ravno they do cook spice :smile:

**Pull request merged**